plugins/login-plugin/src/Components/UIComponents/ClientLogo/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { Image } from "react-native";
+
+const ClientLogo = ({ imageSrc }) => {
+  const [size, setSize] = useState({ width: 0, height: 0 });
+
+  useEffect(() => {
+    Image.getSize(imageSrc, (width, height) => {
+      setSize({ width, height });
+    });
+  }, []);
```

Comment:
This is cool but I think we should keep to the originally designed size and get clients to prepare correct asset so it fits the design.